rdflib/store.py

```diff
@@ -210,7 +210,7 @@ def gc(self):
     def add(
         self,
         triple: Tuple["Node", "Node", "Node"],
-        context: Optional["Graph"],
+        context: Optional[Union["Graph", "Identifier"]],
```

Comment:
```suggestion
        context: Optional["Identifier"],
```
Should accept "Identifier" only I think.

Reply:
Okay, noted. (I'm not up to speed w.r.t. to typing, so it was a bit of a conjecture)